app/ts/constants.ts

```diff
@@ -1,3 +1,35 @@
-export const NETWORKS: { [chainId: string]: { simulationRelay: string, submissionRelay: string, blockExplorer: string, rpcUrl: string }} = {
-	'1': { simulationRelay: 'https://flashbots-cors-proxy.dark-florist.workers.dev/', submissionRelay: 'https://rpc.titanbuilder.xyz', blockExplorer: 'https://etherscan.io/', rpcUrl: 'https://rpc.dark.florist/flipcardtrustone' },
+type Network = { networkName: string, simulationRelay: string, submissionRelay: string, blockExplorer: string, rpcUrl: string, chainId: string, blockExplorerApi: string }
+
+export const MAINNET = {
+	networkName: 'Mainnet',
+	simulationRelay: 'https://flashbots-cors-proxy.dark-florist.workers.dev/',
+	submissionRelay: 'https://rpc.titanbuilder.xyz',
+	blockExplorer: 'https://etherscan.io/',
+	rpcUrl: 'https://rpc.dark.florist/flipcardtrustone',
+	chainId: `0x${1n.toString(16)}`,
+	blockExplorerApi: 'https://api.etherscan.io'
+}
+
+export const NETWORKS = new Map<bigint, Network>([
```

Comment:
Why a `Map` instead of an object?

Reply:
I want to be able to access networks that don't exist (and find out they are undefined)

Reply:
I could use funtypes parsing to achieve that too tho. I am not sure if its any better

Reply:
You can access networks that don't exist on an object too.
```ts
const NETWORKS: { [bigint]: Network } = { ... }
NETWORKS[42] // undefined
NETWORKS[1] // { ... }
```